### DISCUSSION TITLE
What about this video for the Community website banner?

### DISCUSSION THREAD

**Discussion by BOLT04** [EddieHubCommunity/support#600] [Ideas] [answered]
https://www.youtube.com/watch?v=n3OkexlwKtA&lc=UgwZFGalJUlbfxULUrB4AaABAg.9Hyz7j7uEcI9HzRTMnibW2

This short video that Eddie made could be the video we have up in the EddieJaoudeCommunity.github.io website.
The redesign of this web page is being discussed here: https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/issues/106#issuecomment-727628416

Any thoughts on this idea?

**Comment by eddiejaoude** [comment]
I think a video is a great idea 👍 - I have a horizontal version of this video we can use. Or I can create a custom video if people have some ideas of what to have in the background and a script of what I can say 🤓

**Comment by eddiejaoude** [comment, marked as answer]
Video and website created for http://eddiehub.org 🎉
